### ISSUE TITLE
Release v4.2.1 throwing errors

### ISSUE THREAD

**Issue by matthewnessworthy** [imagemin/imagemin-pngquant#21]
Since the release of v4.2.1 the following error is occuring:

```
/home/vagrant/www/node_modules/gulp-util/lib/PluginError.js:73
  if (!this.message) throw new Error('Missing error message');
                     ^

Error: Missing error message
    at new PluginError (/home/vagrant/www/node_modules/gulp-util/lib/PluginError.js:73:28)
    at DuplexWrapper.<anonymous> (/home/vagrant/www/node_modules/gulp-imagemin/index.js:58:8)
    at emitOne (events.js:77:13)
    at DuplexWrapper.emit (events.js:169:7)
    at Through2.<anonymous> (/home/vagrant/www/node_modules/gulp-imagemin/node_modules/imagemin/node_modules/stream-combiner2/node_modules/duplexer2/index.js:49:12)
    at emitOne (events.js:82:20)
    at Through2.emit (events.js:169:7)
    at onwriteError (/home/vagrant/www/node_modules/imagemin-pngquant/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:347:10)
    at onwrite (/home/vagrant/www/node_modules/imagemin-pngquant/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:365:5)
    at WritableState.onwrite (/home/vagrant/www/node_modules/imagemin-pngquant/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:124:5)
```

**Comment by shinnn** [comment]
@matthewnessworthy Can you show us a smallest possible gulpfile by which we can reproduce the error?

**Comment by shinnn** [comment]
@matthewnessworthy No problem. Will fix it soon.

**Comment by shinnn** [comment]
Fixed in v4.2.2.
